libs/shared-config/src/lib/config.ts

```diff
@@ -542,3 +542,12 @@ export const API_URL = process.env.NX_API_URL
 export const S3_UPLOAD_SIGN_URL = process.env.NX_S3_UPLOAD_SIGN_URL
   ? process.env.NX_S3_UPLOAD_SIGN_URL
   : 'http://127.0.0.1:3003/s3/sign'
+
+export const STATUS_LABELS: any = {
```

Comment:
Can we put this on line 535 instead? It's kind of nice to keep the end of this file just urls.

And, in the current context (in a file named "config" in a "shared config" lib), it's not immediately obvious _what kind of_ context we're talking about. Consider prefixing variable name with `MENTORSHIP_MATCH_`.